l10n_it_fatturapa/models/ir_attachment.py

```diff
@@ -92,12 +88,20 @@ def get_xml_string(self):
                     ) % e.args
                 )
 
+        # Amazon sends invalid xml files, so they cannot be detected
+        # using a pattern, we try to parse as asn1, if it fails
+        # we assume is xml
         try:
-            return self.cleanup_xml(self.extract_cades(data))
-        except (ValueError, KeyError) as e:
+            data = self.extract_cades(data)
+        except (ValueError, KeyError):
```

Comment:
@sherpya quindi in caso di XML non firmati, una di queste 2 eccezioni viene sollevata.
Se sai in quale caso una e in quale l'altra, lo puoi aggiungere in un commento?